test-new/e2e/GuardianVerifierBase.t.sol

```diff
@@ -524,14 +576,26 @@ abstract contract OwnableValidatorRecovery_AbstractedRecoveryModule_Base is
             commandParamsForAcceptance[0] = abi.encode(account);
         }
 
-        string memory proofFile = vm.readFile(
-            string.concat(
-                vm.projectRoot(),
-                "/test-new/e2e/email-acceptance-proof.json"
-            )
-        );
+        EmailProof memory emailProof;
+
+        if (guardianType == GuardianType.MockEmailGuardian) {
+            bytes32 nullifier = generateNewNullifier();
 
-        EmailProof memory emailProof = getEmailProof(proofFile);
+            emailProof = generateMockEmailProof(
+                command,
+                nullifier,
+                accountSalt
+            );
+        } else if (guardianType == GuardianType.EmailGuardian) {
```

Comment:
~~is this function (`getAcceptanceEmailProofData`) only called in `acceptGuardian`? Does that mean that `guardianType` is always `GuardianType.MockEmailGuardian` and we would never hit this check?~~ Never mind can see that it is called at beginning of `acceptGuardian`